src/web/wpan-controller/wpan_controller.hpp

```diff
@@ -143,21 +135,26 @@ struct WpanNetworkInfo
 class WPANController
 {
 public:
-    static const struct WpanNetworkInfo *GetScanNetworksInfo(void);
-    static int GetScanNetworksInfoCount(void);
-    static int Start(void);
-    static int Leave(void);
-    static int Form(const char *aNetworkname, int aChannel);
-    static int Scan(void);
-    static int Join(char *aNetworkname, int16_t aChannel, uint64_t aExtPanId,
-                    uint16_t aPanId);
-    static const char *Get(const char *aPropertyName);
-    static int Set(uint8_t aType, const char *aPropertyName, const char *aPropertyValue);
-    static int Gateway(const char *aPrefix, uint8_t aLength, bool aIsDefaultRoute);
-    static int RemoveGateway(const char *aPrefix, uint8_t aLength);
+    const struct WpanNetworkInfo *GetScanNetworksInfo(void);
+    int GetScanNetworksInfoCount(void);
+    const char *GetDBusInterfaceName(void);
+    int Leave(void);
+    int Form(const char *aNetworkname, uint16_t aChannel);
```

Comment:
I just added the Doxygen comments in this file. I will include Doxygen for other files in other PR. Thanks.